Configuration/Applications/python/ConfigBuilder.py

```diff
@@ -928,6 +928,8 @@ def define_Configs(self):
         self.L1MENUDefaultCFF="Configuration/StandardSequences/L1TriggerDefaultMenu_cff"
         self.HLTDefaultCFF="Configuration/StandardSequences/HLTtable_cff"
         self.RAW2DIGIDefaultCFF="Configuration/StandardSequences/RawToDigi_Data_cff"
+        self.RAW2DIGIDataRemapperCFF="Configuration/StandardSequences/RawToDigi_DataRemapper_cff"
```

Comment:
@icali this fragment does not exist

Reply:
and the variable is never used...

Reply:
indeed. This wasn't in my fixup PR, but then it looks like it was reintroduced when commits from icali#1 were redone after merging icali#1